keps/sig-scheduling/2779-support-custom-queue-for-the-scheduler/README.md

```diff
@@ -396,3 +369,11 @@ Why should this KEP _not_ be implemented?
 An alternative is to enhance the current queue to meet some requests. The drawback
 is that we cannot handle all the potential requests as the effort is big, we should
 give the ball to the users, and they can do what they want.
```

Comment:
I disagree with this statement.

I'm more in favor of keeping the interface small with clear use cases.

Reply:
The target of his KEP is to enable scheduler developers to implement their own logic to manage the order the pods. We can expose part of the interface for some known cases now, we will have to expose more interfaces in the future to handle new cases as developers' requests are un-preditable, this needs effort and maybe a new KEP. To expose the whole interface, we can handle the potentail requests by this KEP, needn't extra effort in the future.